python-backend/app/api/nris_services/NRIS_service.py

```diff
@@ -156,15 +155,17 @@ def _process_NRIS_data(data, mine_no):
                 advisories += len(stop_advisories)
                 warnings += len(stop_warnings)
 
+    latest_report = data[0] if data else None
     overview = {
-        'last_inspection': most_recent.get('assessmentDate'),
-        'inspector': inspector,
+        'last_inspection': latest_report.get('assessmentDate') if latest_report else None,
```

Comment:
I could just be derping, but I think `latest_report.get('assessmentDate')` returns `None` if it doesn't exist, so the ternary should be redundant

Reply:
If latest_report is none, calling get() on it will throw an error.